Main.java

```diff
@@ -0,0 +1,18 @@
+package brickBraker;
+
+import javax.swing.JFrame;
+
+public class Main {
+
+	public static void main(String[] args) {
+    Gameplay gamePlay = new Gameplay();		
+    JFrame obj = new JFrame();
+    obj.setBounds(10,10,700,600);
+	obj.setTitle("Breakout Ball");	
```

Comment:
Can the alignment be fixed here? Maybe you can use a Java formatter/indenter.

Reply:
Yes,I will rectify this two issues and push another one